docs/modules/ROOT/pages/object-storage/secrets.adoc

```diff
@@ -0,0 +1,23 @@
+= Object Storage Secret Specification
+
+. Get the required secrets to use your bucket from other applications.
++
+[source,yaml]
+----
+$ oc get secrets objectbucket-creds -oyaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: objectbucket-creds
+data:
+  AWS_ACCESS_KEY_ID: OFk4Qzc0TzFHME9PNjkyWkFGRjU= <1>
+  AWS_REGION: cm1h <2>
```

Comment:
What is this region? Maybe better to use actual values from cloudscale.ch?

Reply:
This looks like base64 encoded? That's how secrets work...

https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/#decoding-secret

Reply:
Ahhhh  😆  I thought it was the code of a region… then it's all good

Reply:
Should we add a note or something about the base64 encoding?